tachyon/math/polynomials/univariate/univariate_sparse_coefficients.h

```diff
@@ -289,16 +294,6 @@ class UnivariateSparseCoefficients {
     return &it->coefficient;
   }
 
-  void RemoveHighDegreeZeros() {  // Fix to RemoveZeros
```

Comment:
Have you tried build and tests? I'm pretty sure by removing this, it will fail because we call this when doing polynomial operations. https://github.com/kroma-network/tachyon/blob/e182278d97c58c934c3d81f00aa710df9c1c5115/tachyon/math/polynomials/univariate/univariate_polynomial_ops.h#L835

Reply:
Yes I did build, test and lint.

Umm that seems to be from old commit? That line is removed in this commit if you look above.